apps/site/assets/css/_event-details.scss

```diff
@@ -39,3 +42,9 @@
     }
   }
 }
+
+.m-event__livestream {
+  width: 100%;
+  height: 56vw;
+  max-height: 560px;
```

Comment:
Does `max-height` have the same value for mobile?

Reply:
Oh good question - I had to check, and yep.  It works totally fine because the video scales to the viewport size.